hawkular-dmr-client/src/main/java/org/hawkular/dmr/api/OperationBuilder.java

```diff
@@ -765,4 +790,9 @@ public T valueAttribute(String value) {
         return new WriteAttributeOperationBuilder<>(null);
     }
 
+    @SuppressWarnings("unchecked")
+    public static GenericOperationBuilder<GenericOperationBuilder<?>> generic(String operationName) {
```

Comment:
this method name is too "generic" :) As a user of this API, I have no idea what this means - and with no javadoc, it isn't clear what it is. I assume by the name of the parameter, this defines the DMR operation name that is to be executed. Or does it create a full DMR ModelNode that is to be used to execute an operation? Couldn't this method be called "operation" or "executeOperation" or "operationName" to be explicit what its for?